packages/next/server/next-dev-server.js

```diff
@@ -121,7 +121,9 @@ export default class DevServer extends Server {
     }
 
     if (!err && res.statusCode === 500) {
```

Comment:
Should this actually be `err === undefined`?

Reply:
It would be more exact but this also allows us to show it for `null`

Reply:
👍